packages/graphql-languageservice/src/LanguageService.ts

```diff
@@ -106,12 +111,7 @@ export class LanguageService {
     _uri: string,
     documentText: string,
     customRules?: ValidationRule[],
-  ) => {
-    if (!documentText || documentText.length < 1) {
-      return [];
-    }
-    return getDiagnostics(documentText, await this.getSchema(), customRules);
-  };
+  ) => getDiagnostics(documentText, await this.getSchema(), customRules);
```

Comment:
Not sure why this was taken out. I'll add it back

Reply:
@ncthbrt maybe try a rebase? probably because you started this PR before I merged an upstream change, oops!